aws/modules/infrastructure_modules/stacks_app/app.tf

```diff
@@ -0,0 +1,28 @@
+# Create DNS Zones
+module "zones" {
+
+  source  = "terraform-aws-modules/route53/aws//modules/zones"
```

Comment:
2 forward slashes here - should only be single

Reply:
<img width="1651" alt="image" src="https://user-images.githubusercontent.com/65091252/165275546-eb95e7c5-5edc-4895-b325-80f076131f64.png">


I guess the child modules for zones are been addressed by // in the orginal git repository
https://github.com/terraform-aws-modules/terraform-aws-route53